src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -286,4 +275,83 @@ void getHearingChannels_ifAdjourmentEnabled_getNextHearing() {
             .containsOnly(TELEPHONE);
     }
 
+    @DisplayName("When adjournment flag is false, returns hearing chanel from the case")
+    @Test
+    void getHearingChannels_ifAdjournmentEnabledAndNextHearingIsNull_returnDefaultHearingChannel() {
+        SscsCaseData caseData = SscsCaseData.builder()
+            .appeal(Appeal.builder()
+                .hearingOptions(HearingOptions.builder()
+                    .wantsToAttend(NO.getValue()).build())
+                        .appellant(Appellant.builder()
+                            .name(Name.builder()
+                                .title("title")
+                                .firstName("first")
+                                .lastName("last").build())
+                        .build())
+                    .build())
+            .dwpIsOfficerAttending(NO.getValue())
+            .adjournment(Adjournment.builder().typeOfNextHearing(null).build())
```

Comment:
This is essentially obsolete (as all adjournment fields will be null), though I understand including if you want to explicitly demonstrate that typeOfNextHearing is null.

Reply:
I have explicitly added for testing to make sure it is null.

Reply:
Removed the unwanted test case